app/components/Button/Button.css

```diff
@@ -89,3 +89,8 @@
   border: none;
   cursor: pointer;
 }
+
+.danger {
+  background: var(--color-red-3);
+  color: var(--color-white);
+}
```

Comment:
Don't we already have a "delete" button styling?

Reply:
We had a danger prop, that did nothing 🤷🏻‍♂️

Reply:
Ahh, wait, the "dark" is red... So dark is actually "danger/delete"
That's...